README.md

```diff
@@ -21,6 +21,35 @@ Requirements
 
 This service is written in [Scala 2.11](http://www.scala-lang.org/) and [Play 2.5](http://playframework.com/), so needs at least a [JRE 1.8](http://www.oracle.com/technetwork/java/javase/downloads/index.html) to run. In addition to Scala and JRE dependencies, the service also depends on [node.js 4.8.4](https://nodejs.org/en/) and [npm 2.15.11](https://www.npmjs.com/).
 
+Steps to start tai-frontend from sources locally
```

Comment:
I don't think the title is very accurate. I think something in lines of "How to install node on Ubuntu" would be better.